src/styles/index.ts

```diff
@@ -587,6 +587,13 @@ const addThemeOverrides = (theme: Theme) => {
         },
       },
     },
+    MuiPaper: {
+      styleOverrides: {
+        root: {
+          boxShadow: `0px 0px 0px 1px ${theme.palette.greyShades[75]}, 0px 1px 0px 0px ${theme.palette.greyShades[75]}`,
```

Comment:
I don't think the second shadow is even visible, seems like you can only keep the first value

Reply:
To make it dark theme compatible, you can also replace an exact hex color with a translucent black color, like `rgba(0, 0, 0, 0.08)` or similar.